ssuslipstream/ssuslipstream.h

```diff
@@ -0,0 +1,29 @@
+/**
+ * @file ssuslipstream.h
+ * @copyright 2015 Jolla Ltd.
+ * @author Thomas Perl <thomas.perl@jolla.com>
+ * @date 2015
+ */
+
+#ifndef SSU_SLIPSTREAM_H
+#define SSU_SLIPSTREAM_H
+
+#include <QObject>
+#include <QSettings>
+#include <QDebug>
+#include <QEventLoop>
+#include <QFile>
```

Comment:
To me it seems that some of these are not even used and some are used in .cpp only remove/move?